### PR TITLE
fix(contributing.md): fixed missing steps inside the CONTRIBUTING.md file.

### PR DESCRIPTION
**Pull request by Pyloris**
## Describe your changes
I added 2 steps in the CONTRIBUTING.md file.
- Added a step to install dependencies for the frontend
- Added a step to tell user about where to find Env Variable values used in deploying nango integration templates.

## Issue ticket number and link
issue no: #2239 
issue link: https://github.com/NangoHQ/nango/issues/2239

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: did not require any tests.
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,6 +16,14 @@ Install the project
 npm i
 ```
 
+Install Frontend Dependencies
+
+```sh
+cd packages/frontend
+npm i
+cd -
+```
```
Comment (bodinsamuel) [inline CONTRIBUTING.md:25]: Frontend is part of the workspaces and does not actually require any dependencies. Do you still have the error that this has fixed?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -58,6 +66,12 @@ nango init
 ```
 
 Change the .env file `NANGO_SECRET_KEY_DEV` and `NANGO_HOSTPORT`.
+
+```sh
+NANGO_SECRET_KEY_DEV="secret key from nango dashboard"
+NANGO_HOSTPORT="host:port"   # can be https://nango.dev or if running locally http://.localhost:3000
+```
```
Comment (bodinsamuel) [inline CONTRIBUTING.md:73]: I'm not sure it's relevant, it's already what the .env is stating by default

Comment (Pyloris) [reply CONTRIBUTING.md:73]: I did not find NANGO_HOSTPORT anywhere in .env, thats why i included it. the `NANGO_SECRET_KEY_DEV` however can be skipped.

Comment (bodinsamuel) [reply CONTRIBUTING.md:73]: the `.env` mentioned here is in `nango-integrations`, when you launch `nango init` it should have created a .env file with all the mentioned var. cf:
https://github.com/NangoHQ/nango/blob/644fc71035dd8d86cd7f9c31fb1eb32649e41801/packages/cli/lib/cli.ts#L239-L250

Comment (Pyloris) [reply CONTRIBUTING.md:73]: you are correct. Mistake on my part. I did not go deep enough. Sorry for the trouble.

**Comment by Pyloris** [comment]
@bodinsamuel please review my google-mail template. I have a PR open for that.

File: CONTRIBUTING.md (hunk 2 above)
Comment (bodinsamuel) [reply CONTRIBUTING.md:73]: no worries ☺️

**Pull request closed without merging**